bindings/protocols/bacnet/bacnet-model.ttl

```diff
@@ -43,7 +43,7 @@ Wherever possible, this model refers to ANSI/ASHRAE Standard 135-2020 for BACnet
 
 hctl:hasTarget rdfs:comment """
 Should be of form bacnet://<device>/<objecttypenr>,<objectinsttr>[/<property>[/<index>]]
-Additional parameters can be specified via URI variable syntax, e.g. ?commandPriority=13 for command priority or ?covIncrement=0.5 for COV increment
+Additional parameters can be specified via URI variable syntax, e.g. ?commandPriority=13 for command priority, ?relinquish for relinquishing a priority or ?covIncrement=0.5 for COV increment
```

Comment:
I know this isn't markdown, but the backticks provide additional clarity even when they are displayed and not transformed into `<code>` or similar markup.
```suggestion
Additional parameters can be specified via URI variable syntax, e.g., `?commandPriority=13` to change command priority, `?relinquish` to relinquish a priority, or `?covIncrement=0.5` to increment COV.
```